src/Kanbas/store/enums/eTrueOrFalse.ts

```diff
@@ -0,0 +1,4 @@
+export enum eTrueOrFalse {
```

Comment:
rename to `eBooleanChoices` and extend `eChoices`. We can keep the generic methods like the ones in src/Kanbas/Courses/Quizzes/Preview/Question/Common/MultipleChoice/index.tsx in `eChoices`.

Reply:
Typescript doesn't offer extend, unless you want me to create union of two enums to support that extends functionality?

Reply:
No, that would overcomplicate it. Let's keep it as it is.